### PR TITLE
Sprint 37 TLT-2249 Stop html escaping some course info fields

### PR DESCRIPTION
**Pull request by rascalking**
Per devops, we support embedded HTML in the description and notes fields.

Also, stop embedding the `description` and `notes` fields in a `textarea` in the widget, so the users get a better idea of what the final result will look like.

### CODE REVIEW

**Comment by elliottyates** [comment]
@elliott

File: course_info/templates/course_info/editor.html [hunk 1]
```diff
@@ -75,11 +75,7 @@
                       <strong>{{ field.label }}:</strong>
                     </label>
                     {% if field.value %}
-                      {% if field.key in textarea_fields %}
-                        <textarea class="form-control" rows="1" disabled>{{ field.value }}</textarea>
-                      {% else %}
-                        {{ field.value }}
-                      {% endif %}
+                      {{ field.value }}
```
Comment (elliottyates) [inline course_info/templates/course_info/editor.html:78]: Maybe we can also remove line 196 as well then?

**Comment by elliottyates** [comment]
:+1:

File: course_info/templates/course_info/editor.html (hunk 1 above)
Comment (rascalking) [inline course_info/templates/course_info/editor.html:78]: Done.

**Pull request merged**